pkgs/development/python-modules/tensorflow/bin.nix

```diff
@@ -113,7 +117,6 @@ in buildPythonPackage {
         -e "s/Requires-Dist: numpy (.*)/Requires-Dist: numpy/"
     )
     wheel pack ./unpacked/tensorflow*
-    mv *.whl $orig_name # avoid changes to the _os_arch.whl suffix
```

Comment:
Apparently this was introduced specifically for darwin: https://github.com/NixOS/nixpkgs/pull/178535/commits/6a2dad3a376454f3979982a0e00aa2d89fe55fef

Reply:
Is it the intention for `tensorflow-bin` to support multiple versions of tensorflow in a single nixpkgs revision (e.g. through overrides)? I see the attrset entries contain a "version" field. This change might be breaking non-default instantiations of tensorflow-bin, maybe we should test them and if they fail - make the `mv` conditional rather than remove it

Reply:
Well, as it was introduced as a "fix" for darwin (which it probably was at the time), I assume than we can remove it now... as it is the cause of this new failure.
What is your take on this @veprbl ?